### ISSUE TITLE
Cancelling last few games rendered every game cancelled

### ISSUE THREAD

**Issue by Karakaii** [empiricaly/meteor-empirica-core#241]
I had 200 games, with complete randomisation and four different treatments (50 each), Two games had participants who had started but then went away without finishing the games. All the other games were finished. I cancelled these last two games. When I cancelled the last one, it made every game count as cancelled. In the data I have every exitStatus as gameCancelled. (Good thing I saved a final version of the data before that). 

Windows, Firefox, Empirica:core 1.15.3

**Comment by npaton** [comment]
We just fixed this issue in 1.15.4 2 days ago. From your description we were not sure if the finished games got a "cancelled" `status`. From the rest of your description and our tests, we don't think so. But indeed, the players' `exitStatus` would be incorrectly set to cancelled previously. This should no longer be the case in the latest version.

We tested on 1.15.3 and could reproduce the issues and then tested in 1.15.4, and the issue is fixed 👍
